src/main/java/org/folio/processing/mapping/mapper/writer/marc/MarcRecordModifier.java

```diff
@@ -126,6 +127,7 @@ public void processUpdateMappingOption(List<MarcMappingDetail> marcMappingRules)
       return;
     }
 
+    var notUpdatedDataFields = newLinkedList(incomingMarcRecord.getDataFields());
```

Comment:
Just interesting, not a suggestion. Did you used this method only for code beauty?
```suggestion
    var notUpdatedDataFields = new LinkedList<>(incomingMarcRecord.getDataFields());
```

Reply:
Yes, looks nice for me